build-tests/install-test-workspace/workspace/typescript-v3-test/package.json

```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
     "@rushstack/eslint-config": "*",
     "@rushstack/heft": "*",
-    "typescript": "~4.8.4",
+    "typescript": "~5.0.4",
```

Comment:
This is wrong. This project was meant to test the oldest supported TypeScript compiler version.